qutip/entropy.py

```diff
@@ -223,10 +224,8 @@ def entropy_mutual(rho, selA, selB, base=e, sparse=False):
     return out
 
 
-def _entropy_relative(rho, sigma, base=e, sparse=False):
+def entropy_relative(rho, sigma, base=e, sparse=False):
```

Comment:
Sparse kwarg isn't documented - just needs mentioning that it's about the eigensolver method.  (If you're at it, you could also make the `qobj` types probably crossrefs too.)

Reply:
Documentation for sparse and links to Qobj added.